cypress/support/commands.js

```diff
@@ -1,13 +1,18 @@
 const USER_ROLES_MAP = {
-    "User": "",
-    "Team Lead": "role=leads",
-    "Manager": "role=managers",
-    "Accountant": "role=accountants",
-    "Admin": "role=admins"
-}
-
-Cypress.Commands.add('loginAs', (role) => {
-    window.localStorage.setItem('USER_INFO', '{"id":1,"firstName":"Demo","lastName":"User"}')
-    window.localStorage.setItem('ACCESS_TOKEN', `userId=1&${USER_ROLES_MAP[role]}`)
-})
+  User: '',
+  'Team Lead': 'role=leads',
+  Manager: 'role=managers',
+  Accountant: 'role=accountants',
+  Admin: 'role=admins'
+};
 
+Cypress.Commands.add('loginAs', role => {
+  window.localStorage.setItem(
+    'USER_INFO',
+    '{"id":32,"firstName":"Mindaugas","lastName":"Maceika"}'
+  );
```

Comment:
not sure if it is actually more readable and for me it is even more confusing to read this way

Reply:
It's my auto formatting settings, formatting was changed back.